browser/brave_profile_prefs.cc

```diff
@@ -377,13 +371,6 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   NTPBackgroundPrefs::RegisterPref(registry);
 #endif
 
-#if BUILDFLAG(ETHEREUM_REMOTE_CLIENT_ENABLED)
-  registry->RegisterIntegerPref(kERCPrefVersion, 0);
-  registry->RegisterStringPref(kERCAES256GCMSivNonce, "");
-  registry->RegisterStringPref(kERCEncryptedSeed, "");
-  registry->RegisterBooleanPref(kERCOptedIntoCryptoWallets, false);
```

Comment:
I would rather not deprecate these prefs in the normal way and clear them in case someone needs to revert to an old version of Brave and somehow get access to this. LMK what you think though @darkdh

Reply:
Downgrading browser version on same profile is not supported and is not guaranteed to work afaik.

Let's keep these values in profile prefs for some time(1 year), but do it explicitly:
- Prefs are still registered.
- Renamed to smth like `kERCPrefVersion` -> `kERCPrefVersionDeprecated` with a comment of current month/year
So there is neither risk someone reuses name of some of these prefs, nor chromium changes something so unregistered prefs get automatically cleared.

If a user appears with complains, we can think of some way of recovering based on these saved prefs.

Reply:
I know this is really hacky, but we could keep a tool around in a repo for seed phrase recovery. That way if the user still has the prefs set on disk, they can still access them from the tool to migrate to an updated version. Would be useful in the case of old password migrations that we have for wallets too.

Reply:
Anthony and I talked about exactly this but only if the need came up in support which we don't expect it to.

Reply:
I added these prefs into brave wallet prefs and renamed them to deprecated here a6bc6a11365459749e34a6fffde273e909fee495